### PR TITLE
Use enhanced for loops where possible

### PR DESCRIPTION
**Pull request by larsgrefer**

### CODE REVIEW

File: java/jakarta/el/Util.java [hunk 1]
```diff
@@ -380,11 +380,12 @@ static Method findMethod(Class<?> clazz, Object base, String methodName,
...
             StringBuilder sb = new StringBuilder();
-            for (int i = 0; i < types.length; i++) {
-                if (types[i] == null) {
+            for (Class<?> type : types) {
+                if (type == null) {
                     sb.append("null, ");
-                } else {
-                    sb.append(types[i].getName()).append(", ");
+                }
+                else {
```
Comment (markt-asf) [inline java/jakarta/el/Util.java:387]: These formatting changes need to be removed from this PR. Tomcat coding convention is to use `} else {` on a single line.

Comment (larsgrefer) [reply java/jakarta/el/Util.java]: The if-else formatting should be fixed now

**Comment by markt-asf** [comment]
Thanks for the updates. I'm going to merge this via the command line as there are a few minor things I want to tweak - mainly variable names.

**Comment by markt-asf** [comment]
Applied manually with some minor changes (and a larger follow-on commit)

**Pull request closed without merging**